dvc/commands/daemon.py

```diff
@@ -35,6 +35,12 @@ def run(self):
         return 0
 
 
+class CmdDaemonTest(CmdDaemonBase):
+    def run(self):
+        with open(self.args.file, "w", encoding="utf8") as f:
+            f.write("Data Version Control\n")
```

Comment:
I'm not sure I understand the point of this. A special command to just write some string to a file?

Reply:
I'm not against test helpers in general, just wondering about this particular one. Tempted to think about something more useful that we could use instead, but can't think of anything just yet.